operators/src/pro/cache/cache_chunks.rs

```diff
@@ -186,11 +174,28 @@ where
         // So the query is not updated.
         Ok(())
     }
+
+    fn cache_element_hit(&self, query: &Self::Query) -> bool {
+        let temporal_hit = if let Some(time_bounds) = self.time_interval {
+            time_bounds == query.time_interval || time_bounds.intersects(&query.time_interval)
```

Comment:
shouldn't it be `contains`?

Reply:
No this is the check if a single element should be part of the query answer.